src/NUnitFramework/framework/Constraints/ExactCountConstraint.cs

```diff
@@ -43,12 +53,10 @@ public class ExactCountConstraint : PrefixConstraint
         public ExactCountConstraint(int expectedCount, IConstraint itemConstraint)
             : base(itemConstraint)
         {
-            this.expectedCount = expectedCount;
-            this.DescriptionPrefix = expectedCount == 0
-                ? "no item"
-                : expectedCount == 1
-                    ? "exactly one item"
-                    : string.Format("exactly {0} items", expectedCount);
+            Guard.ArgumentNotNull(itemConstraint, "itemConstraint");
```

Comment:
> I suppose somebody has to create a proposal and shepherd it through them if that's going to happen.

Ok - I'll bite. ;-)

I've posted a thread on NUnit Developers, as I think it's time for a full cross-project decision, and I think this is something that would be advantageous to discuss in a forum accessible to all team members.

https://groups.google.com/forum/#!topic/nunit-developer/m00Oeo9bSqA